crates/codegen/syntax/src/combinator_tree.rs

```diff
@@ -65,16 +65,15 @@ impl<'context> CombinatorTree<'context> {
 
```

Comment:
> - [x] Arithmetic operations revert on underflow and overflow. You can use unchecked { ... } to use the previous wrapping behaviour.

I see you marked this as **Not Syntactic**. Not sure I'm following. The syntax `unchecked {..}` should be illegal before that version. Right?

Reply:
I had thought that unchecked was valid before that, but you are right, it is new syntax.

Reply:
done